charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -29,6 +29,9 @@ spec:
     metadata:
       labels:
         installer-job: {{ .name }}
+        {{- if .root.Values.global.jdbc.podbinding }}
+        aadpodidbinding: {{ .root.Values.global.jdbc.podbinding }}
+        {{- end  }}
```

Comment:
I guess for the installer there is no option to add labels from values. Probably its a good candidate to expose the option.